spec/controllers/cavc_remands_controller_spec.rb

```diff
@@ -65,23 +85,20 @@
         end
       end
 
-      context "with correct parameters" do
-        it "creates the CAVC remand and new appeal" do
-          remand_count = CavcRemand.count
-          cavc_count = Appeal.court_remand.count
-          subject
-
-          expect(response.status).to eq(201)
-          response_body = JSON.parse(response.body)
+      context "when sub-type is JMR with correct parameters" do
+        include_examples "creates a remand depending on the sub-type"
+      end
 
-          expect(response_body["cavc_remand"]["source_appeal_id"]).to eq(source_appeal.id)
-          expect(response_body["cavc_remand"]["decision_issue_ids"]).to match_array(decision_issue_ids)
-          expect(CavcRemand.count).to eq(remand_count + 1)
+      context "when sub-type is MDR" do
+        let(:remand_subtype) { Constants::CAVC_REMAND_SUBTYPES.keys.third }
+        context "with unrequired judgement and mandate date parameters" do
```

Comment:
Less confusing to me -- and to @n1zyy who is pairing as I review this ;)
```suggestion
        context "with judgement and mandate date parameters" do
```